apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -683,13 +683,19 @@ export class WorkshopForm extends React.Component {
 
   shouldConfirmSave() {
     const workshop = this.props.workshop;
+
     if (!workshop || workshop.enrolled_teacher_count === 0) {
       return false;
     }
+
+    // If location address is modified, then returned to blank,
+    // this.state.location_address is a blank string instead of null.
     return (
       this.state.sessionsModified ||
       this.state.location_name !== workshop.location_name ||
-      this.state.location_address !== workshop.location_address ||
+      (this.state.location_address === ''
+        ? null
+        : this.state.location_address) !== workshop.location_address ||
```

Comment:
Tip:  Since empty string is falsy, you could get rid of the ternary and express this clause as

```js
(this.state.location_address || null) !== workshop.location_address
```

Reply:
Probably overkill, but this would result in the dialog not being shown if a user put in other falsy values like 0 for location address, right?

Reply:
It shouldn't, since the user can only input a string for location_address, and all strings except for the empty string are truthy:
![image](https://user-images.githubusercontent.com/1615761/83285063-79dc4f80-a192-11ea-8f0a-98295c362304.png)

Reply:
I was able to save "0" as a location_address 😬